pkg/logcli/client/client.go

```diff
@@ -178,6 +179,12 @@ func (c *DefaultClient) doRequest(path, query string, quiet bool, out interface{
 		req.Header.Set("X-Scope-OrgID", c.OrgID)
 	}
 
+	//The token presence overrides basic-auth authentication
+	if c.Token != "" {
+		req.Header.Set("Authorization", "Bearer "+c.Token)
+		req.Header.Add("Accept", "application/json")
```

Comment:
Why does the `Accept` header get set here?

Reply:
logcli does not set this header elsewhere. By setting it here, we'd expect a 406 error if the response is not adequate (`! application/json"`). 
IMHO it should be set for all request but this PR's scope is to add support for bearer tokens.

Reply:
Then leave it out of this PR entirely?